twilio-sheet/README.md

```diff
@@ -0,0 +1,104 @@
+    1. Visit the web app’s script
+    1. Enter the URL of your spreadsheet at the top.
+
+1. Publish web application
+    1. Click Publish > Deploy as a web app > Deploy (leave all the default
+       options selected).
+    1. You will be given a URL, copy it and paste in a new tab in your browser.
+    1. This will bring up the web app that looks like this (I did not make it public so I am sharing the screenshot):
+
+    ![Screenshot of phone number web app](https://cdn.jsdelivr.net/gh/TechandEco/GSuite-samples@master/twilio-sheet/webapp-screenshot.png)
+
+1. Return to web app’s script on the Code.js file
+    - Select from the function drop-down “doGet” and click the Run button (►).
+
+1. Return to your spreadsheet
+    1. Click Tools > Script editor
+    1. At the top of the code enter the following Twilio Account ID, Auth
+       token, and the phone number you purchased from Twilio.
+1. Enter data via the web app
+    - Visit the tab with the web app and enter your personal phone number and
+       click “Submit”, the one you used to verify your account with Twilio
+       (this is not the phone number you bought from Twilio to send an SMS).
+    ![Screenshot of Twilio values in script](https://cdn.jsdelivr.net/gh/TechandEco/GSuite-samples@master/twilio-sheet/twilio_account_info_screenshot.png)
+
+1. Return to the spreadsheet and you will see your phone number and the
+    time you entered it logged in the sheet.
+    > Optional if you do not wish to enter phone numbers via the web app you
+    > can directly enter the phone numbers manually in the “Phone number”
+    > column.This is possible since the web app simply appends new data off
+    > the last row of data entered in the sheet, so it dynamically adapts.
+
+1. Prepare message and authorize the script to send SMS
+    1. Refresh Google Sheet
+    1. Enter a message in the “Message” column (ex: “Hi this is a message”)
+    1. A new menu at the top of the sheet called “Send SMS”  should have been
+       displayed after a few seconds after refreshing the sheet.
+    1. Click Send SMS menu > Send to All
+    1. When prompted, click the "Review permissions" and click "Allow" so the
+       script can send texts on your behalf.
+        > Note: If you get a warning that "This app isn't verified,"
+        > continue with the verification process by clicking "Advanced" and
+        > then scroll down and click the grey text at the bottom that begins
+        > with "Go to..."
+    1. A text message should have been sent to your phone and the “Status” column should have been updated with “Sent” and the date.
+
+1. [OPTIONAL] Continue to send messages moving forward.
+    1. Via Twilio trial
+        - If you wish to send more messages via the Twilio trial, you can only do so to your personal phone. You can continue to enter that number via the web app or enter it manually in the sheet and simply change the message in the “Message” column to something new.
```

Comment:
Nit: it's easier to view file differences in Markdown if you start every new sentence in its own line, just make sure it's indented correctly.